pkg/install.go

```diff
@@ -10,20 +10,22 @@ import (
 	"io"
 	"net/http"
 	"os"
-	"os/exec"
 	"path/filepath"
 	"runtime"
 	"strconv"
 	"strings"
 
 	"github.com/projectdiscovery/gologger"
+	ospath "github.com/projectdiscovery/pdtm/pkg/path"
 )
 
 var extIfFound = ".exe"
 
 // Install installs given tool at path
-func Install(tool Tool, path string) error {
-	if path, _ := exec.LookPath(tool.Name); path != "" {
+func Install(path string, tool Tool) error {
+	executablePath, exists := ospath.GetExecutablePath(path, tool.Name)
```

Comment:
Not really an issue but worth mentioning, tipically inline syntax is preferred if you don't need `executablePath` later:
```go
if executablePath, exists := ospath.GetExecutablePath(path, tool.Name); exists {
...
}
```